src/django/sqls/0145_remove_items_without_matches_and_related_facilities.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE PROCEDURE remove_items_without_matches_and_related_facilities()
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    item_ids INTEGER[];
+BEGIN
+    SELECT array_agg(id) INTO item_ids
+    FROM api_facilitylistitem 
+    WHERE NOT EXISTS (
+        SELECT 1
+        FROM api_facilitymatch
+        WHERE api_facilitymatch.facility_list_item_id = api_facilitylistitem.id
+    )
+    AND NOT EXISTS (
+        SELECT 1
+        FROM api_facility
+        WHERE api_facility.created_from_id = api_facilitylistitem.id
```

Comment:
I think, this way you could accidentally remove facilitylistitem that have been just inserted into table: they don'y have matches yet, they they don't have facility_id. So, I this condition needs to be revised

Reply:
Probably we should also check the **status** of api_facilitylistitem. It shouldn't be equal to -> _UPLOADED, PARSED or GEOCODED_
cc @killev @mazursasha1990

Reply:
Sounds reasonable

Reply:
I added additional conditions.